src/lib/serviceRoutines/putIndividualContextEntity.cpp

```diff
@@ -39,6 +40,22 @@
 /* ****************************************************************************
 *
 * putIndividualContextEntity - 
+*
+* Corresponding Standard Operation: UpdateContext/UPDATE
+*
+* PUT /v1/contextEntities/{entityId::id}
+* PUT /ngsi10/contextEntities/{entityId::id}
+*
+* Payload In:  UpdateContextElementRequest
+* Payload Out: UpdateContextElementResponse
+*
+* URI parameters:
+*   - attributesFormat=object
```

Comment:
Wait, this is not about func-tests but the actual implementation ...
And you are right, these filters must be taken care of right here, to create the Standard Op UpdateContextRequest.